parquet/src/test/java/org/apache/iceberg/parquet/TestParquetDeleteWriters.java

```diff
@@ -74,7 +74,7 @@ public void createDeleteRecords() {
 
   @Test
   public void testEqualityDeleteWriter() throws IOException {
-    File deleteFile = temp.newFile();
+    File deleteFile = temp;
```

Comment:
we can probably remove this variable

Reply:
should this actually not call createTempFile(temp)?

Reply:
@TempDir annotation creates a new file for each test. In a test if we want to create multiple files, then createTempFile would be of use.

Reply:
as I mentioned in another comment, this seems to be inconsistent across the PR on whether we create a separate new temp file

Reply:
The tests in TestParquetDeleteWriters.java expects the file to be actually created while the tests in TestParquet don't want the file to be present on the system, as it would throw file already exists exception

Reply:
the tests in TestParquetDeleteWriters.java expects the file to be actually created while the tests in TestParquet don't want the file to be present on the system, as it would throw file already exists exception